java/src/main/java/de/stroeer/api/config/JacksonConfiguration.java

```diff
@@ -1,35 +0,0 @@
-package de.stroeer.api.config;
-
-import static java.util.Collections.singletonList;
```

Comment:
Fände es aber doof wenn die Clients dann alle copy Pasta machen. Wie stehst du dazu zwei packages zu publishen und nur wer die jackson serializer braucht nutzt die zweite dependency.

Reply:
I think we should consider [transcoding](https://cloud.google.com/endpoints/docs/grpc/transcoding#deploying_a_service_that_uses_transcoding) instead! [Envoy](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/grpc_json_transcoder_filter) or [grpc-proxy](https://github.com/grpc-ecosystem/grpc-gateway) support this.